data/presets/shop/hairdresser.json

```
@@ -7,6 +7,10 @@
     "terms": [
         "beard"
     ],
+    "fields": [
+        "name",
+        "gender"
+    ],    
```

there's some superfluous whitespace here
```suggestion
    ],
```